api/proto/storage/sql.proto

```diff
@@ -0,0 +1,40 @@
+syntax = "proto3";
+package proto.storage;
+
+option go_package = "github.com/spinnaker/kleat/api/client/storage";
+
+import "google/protobuf/wrappers.proto";
+
+message SQL {
```

Comment:
I see that the `Redis` and `SQL` protos aren't actually consumed by any microservice config protos yet -- could you remind me if they are only consumed by Keel and the plan is to add Keel support in a followup PR?

Reply:
Keel requires SQL. you are right that Redis isnt immediately needed. got rid of it.

Reply:
Good point. I know that SQL can be used as an alternative to Redis for other microservices but I am not sure how well tested or supported it is. Maybe creating an issue to track the inconsistency is a better short term strategy. In our settings we use SQL only for Keel and have Redis as the datastore for other microservices. If you want, I can drop an issue for SQL vs Redis in other microservices.

Reply:
I think it's well-supported and might even be the recommended backend for Clouddriver and Orca at this point, but at least for Clouddriver there are other [parameters](https://spinnaker.io/setup/productionize/persistence/clouddriver-sql/#configure-clouddriver-to-use-mysql) that we'd need to add before fully supporting it in Kleat, and I don't want to block your addition of Keel support on that. If you want to add a TODO here or make a GH issue just to make it clear to future contributors that more work is needed to configure the SQL backend for the other microservices, that would be 💯 !

Reply:
done. Added issue https://github.com/spinnaker/kleat/issues/144

Reply:
Thanks so much, @nimakaviani, I really appreciate it! I'll keep an eye out for your next PR adding Keel support!